CHANGELOG.md

```diff
@@ -25,6 +25,7 @@
 - Server profile
 - Server profile template
 - Uplink set
+- Volume attachment
```

Comment:
mark it as storage volume attachment.

Reply:
This resource's provider name is volume_attachment and the previous releases also used the same name (volume attachment)